xds/third_party/googleapis/import.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# Copyright 2020 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Update VERSION then in this directory run ./import.sh
+
+set -e
+BRANCH=master
+VERSION=ca1372c6d7bcb199638ebfdb40d2b2660bab7b88
+GIT_REPO="https://github.com/googleapis/googleapis.git"
+GIT_BASE_DIR=googleapis
+SOURCE_PROTO_BASE_DIR=googleapis
+TARGET_PROTO_BASE_DIR=src/main/proto
+FILES=(
+google/api/expr/v1alpha1/syntax.proto
+)
+
+pushd `git rev-parse --show-toplevel`/xds/third_party/googleapis
+
+# clone the envoy github repo in a tmp directory
```

Comment:
nit: s/envoy/googleapis

Reply:
Do you mean I need to change `# clone the envoy github repo in a tmp directory` to `# clone the s/envoy/googleapis github repo in a tmp directory` here?

Reply:
I meant to replace "envoy" with "googleapis" since this is cloning the googleapis repo.
`# clone the googleapis github repo in a tmp directory` should be fine.

Reply:
Got it! Thanks for the clarification :)